test/analysis/test_summary.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import pandas as pd
+import os
+from glob import glob
+from src.analysis.summary.summary import summarize_networks
+
+# Notes:
+# - Column labels are required
+# - 'NODEID' is required as the first column label
+# - file_paths must be a list, even if a single file path is passed
+
+class TestSummary:
```

Comment:
These tests pass in GitHub Actions but fail when I test them locally.  I wonder if it could be because of the file path differences across operating systems?  We should decide whether to make the comparison to the expected summary file more robust or skip this test on Windows, which we already do elsewhere in the SPRAS test suite.